src/app/not-found.tsx

```diff
@@ -0,0 +1,34 @@
+import Image from 'next/image';
+import Buttons from '@/components/SharedComponents/Buttons';
+import { Lexend_Giga } from 'next/font/google';
+const inter = Lexend_Giga({ subsets: ['latin'] });
+
+export default function NotFound() {
+	return (
+		<main className="flex min-h-screen flex-col items-center justify-between">
```

Comment:
for this class name maybe use 
`<main className="flex min-h-screen flex-col items-center justify-between pt-5">`